docs/pages/tutorial/deploy-to-mainnet/main.md

```diff
@@ -0,0 +1,172 @@
+## Sepolia / Mainnet Deployment
+
+This is a guide to deploy a Dojo world to Sepolia.
+The steps for Mainnet are exactly the same, just replace the chain name and ID when necessary.
+
+
+### Setup
+
+* You need a [Starknet RPC Provider](https://www.starknet.io/fullnodes-rpc-services/) to deploy contracts on-chain. After you get yours, check if it works and is on the chain you want to deploy (`SN_SEPOLIA` or `SN_MAIN`)
+
+```sh
+# run this...
+curl --location '<RPC_PROVIDER_URL>' \
+--header 'Content-Type: application/json' \
+--data '{"id": 0,"jsonrpc": "2.0","method": "starknet_chainId","params": {}}'
+
+# you should get an output like this...
+{"jsonrpc":"2.0","result":"0x534e5f5345504f4c4941","id":0}
+
+# now paste the hex result part on this command... 
+echo 0x534e5f5345504f4c4941 | xxd -r -p
+
+# which !must! output SN_SEPOLIA or SN_MAIN
+SN_SEPOLIA
+```
+
+* Declare the `sepolia` profile in [`Scarb.toml`](https://github.com/rsodre/512karat/blob/main/dojo/Scarb.toml)
+
+```toml
+[profile.sepolia]
+```
+
+* Create the [`dojo_sepolia.toml`](https://github.com/rsodre/512karat/blob/main/dojo/dojo_sepolia.toml) dojo config file, with the same contents of [`dojo_dev.toml`](https://github.com/rsodre/512karat/blob/main/dojo/dojo_dev.toml), except for `[env]`, in which we're going to expose the `world_address` only:
+
+```toml
+[env]
+# rpc_url = ""         # env: STARKNET_RPC_URL
+# account_address = "" # env: DOJO_ACCOUNT_ADDRESS
+# private_key = ""     # env: DOJO_PRIVATE_KEY
+world_address = "0x30ce813d2c4d55298764d676bbc1f37fb8b6337e29337692c7133f52d106878"
```

Comment:
The world address could be commented if it's the first deploy. But once deployed, it's always recommended to have it yeah.